src/app/modules/main-module/components/main/main.component.ts

```diff
@@ -8,20 +8,21 @@ import { GameService } from 'src/app/services/game.service';
   styleUrls: ['./main.component.scss'],
 })
 export class MainComponent implements OnInit {
+
```

Comment:
comprobad estas cosas antes de subir, si vais a commitear un fichero por espacios simplemente....no lo hagáis